storage/libradb/src/lib.rs

```diff
@@ -304,18 +304,17 @@ impl LibraDB {
             .version())
     }
 
-    /// Returns the latest ledger infos per epoch starting with the given epoch num:
-    /// - the latest ledger info of the current epoch is just the last ledger info in the system
-    /// - the latest ledger infos of previous epochs contain reconfiguration validator sets.
+    /// Returns ledger infos reflecting epoch bumps starting with the given epoch.
+    ///
     /// Returns error in case `start_epoch` is higher than the currently known epoch.
```

Comment:
This is probably not true.

Reply:
yeah.. but I didn't touch it.. probably removing..